core/unit_test/TestViewAPI.hpp

```diff
@@ -837,18 +837,15 @@ struct TestViewMirror {
                                                     view_const_cast(v));
   }
 
-  template <class MemoryTraits, class Space>
+  template <class View>
   struct CopyUnInit {
-    using mirror_view_type = typename Kokkos::Impl::MirrorViewType<
-        Space, double *, Layout, Kokkos::HostSpace, MemoryTraits>::view_type;
-
-    mirror_view_type a_d;
+    View a_d;
```

Comment:
Maybe add a `static_cast(Kokkos::is_view_v<View>, "Expecting a Kokkos::View")` ?

Reply:
This is a test functor used in only one test. I don't think we need extra security.